### PR TITLE
[Do not Merge] Triage tool two factor changes

### PR DESCRIPTION
**Pull request by siva-muruganandham**

### CODE REVIEW

File: DFC.Common.SharedContent.Pkg.Netcore/Model/Response/TraigeLevelTwoResponse.cs
Comment (ZhaomingLu2009) [inline DFC.Common.SharedContent.Pkg.Netcore/Model/Response/TraigeLevelTwoResponse.cs:1]: Please tidy up the empty spaces

File: DFC.Common.SharedContent.Pkg.Netcore/Infrastructure/Strategy/TriageLeveTwoQueryStrategy.cs [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+public class TriageLeveTwoQueryStrategy : ISharedContentRedisInterfaceStrategyWithRedisExpiry<TraigeLevelTwoResponse>
+{
+    private readonly IGraphQLClient client;
+    private readonly ILogger<TriageLeveTwoQueryStrategy> logger;
+
+    public TriageLeveTwoQueryStrategy(IGraphQLClient client, ILogger<TriageLeveTwoQueryStrategy> logger)
+    {
+        this.client = client;
+        this.logger = logger;
```
Comment (ZhaomingLu2009) [inline DFC.Common.SharedContent.Pkg.Netcore/Infrastructure/Strategy/TriageLeveTwoQueryStrategy.cs:16]: Is this logger been used?

File: DFC.Common.SharedContent.Pkg.Netcore/Model/Response/TraigeLevelTwoResponse.cs
Comment (siva-muruganandham) [reply DFC.Common.SharedContent.Pkg.Netcore/Model/Response/TraigeLevelTwoResponse.cs:1]: Done

File: DFC.Common.SharedContent.Pkg.Netcore/Infrastructure/Strategy/TriageLeveTwoQueryStrategy.cs (hunk 1 above)
Comment (siva-muruganandham) [reply DFC.Common.SharedContent.Pkg.Netcore/Infrastructure/Strategy/TriageLeveTwoQueryStrategy.cs:16]: Added logging

File: DFC.Common.SharedContent.Pkg.Netcore/Model/Common/TriageLevelOne.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
+﻿using Newtonsoft.Json;
+using System.Text.Json.Serialization;
+
+namespace DFC.Common.SharedContent.Pkg.Netcore.Model.Common
+{
+
```
Comment (ZhaomingLu2009) [inline DFC.Common.SharedContent.Pkg.Netcore/Model/Common/TriageLevelOne.cs:6]: Please remove this empty line

Comment (siva-muruganandham) [reply DFC.Common.SharedContent.Pkg.Netcore/Model/Common/TriageLevelOne.cs]: done

File: DFC.Common.SharedContent.Pkg.Netcore/Infrastructure/Strategy/TriageLeveTwoQueryStrategy.cs [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    }
+
+    public async Task<TraigeLevelTwoResponse> ExecuteQueryAsync(string key, string filter, double expire = 4)
+    {
+        logger.LogInformation("TriageLeveTwoQueryStrategy -> ExecuteQueryAsync");
+        string query = @$"
+           query MyQuery {{
+          triageLevelTwo(status: {filter}) {{
```
Comment (ZhaomingLu2009) [inline DFC.Common.SharedContent.Pkg.Netcore/Infrastructure/Strategy/TriageLeveTwoQueryStrategy.cs:24]: Please tidy up this query string and remove unused empty spaces

Comment (siva-muruganandham) [reply DFC.Common.SharedContent.Pkg.Netcore/Infrastructure/Strategy/TriageLeveTwoQueryStrategy.cs]: done

**Review by ZhaomingLu2009 — APPROVED** [review]
Reviewed

**Pull request closed without merging**